src/css/profile/mobile/common/datepicker.less

```diff
@@ -0,0 +1,417 @@
+.tau_datepicker_material {
```

Comment:
Use the kebab case instead of the snake.

Reply:
I think it would be good to remove the 'tau_' prefix if there is a reason.

Reply:
done.